spec/factories.rb

```diff
@@ -44,6 +48,31 @@
         end
       end
     end
+
+    trait :with_dual_application do
+      scheme { 'B' }
+      application_form { create(:completed_application_form, application_choices: [create(:submitted_application_choice)]) }
+      ucas_status { :awaiting_provider_decision }
+    end
+
+    trait :with_multiple_acceptances do
+      scheme { %w[D U] }
+      application_form do
+        create(:completed_application_form,
+               application_choices: [create(:application_choice, :with_accepted_offer), create(:application_choice, :with_accepted_offer)])
```

Comment:
Do we need two application choices on this factory? Isn't the idea that there's one choice in Apply and one in UCAS?

Reply:
Yeah, that's not quite right. I will amend the after build block.